File: back/DevC/src/main/java/com/back/devc/domain/interaction/postLike/dto/PostLikeResponse.java
package com.back.devc.domain.interaction.postLike.dto;

/**
 * 좋아요 추가/취소 응답 DTO
 */
public record PostLikeResponse(
        Long postId,
        boolean liked,
        long likeCount,
        String message
) {
}

File: back/DevC/src/main/kotlin/com/back/devc/domain/interaction/postlike/dto/PostLikeResponse.kt
package com.back.devc.domain.interaction.postLike.dto

/**
 * 좋아요 추가/취소 응답 DTO
 */
data class PostLikeResponse(
    val postId: Long,
    val liked: Boolean,
    val likeCount: Long,
    val message: String,
)